src/main/java/com/hillel/items_exchange/service/UserService.java

```diff
@@ -57,4 +64,32 @@ public boolean existsByEmail(String email) {
     private UserDto mapUserToDto(User user) {
         return modelMapper.map(user, UserDto.class);
     }
+
+    public List<ChildDto> getChildren(User parent) {
+        return convertToDto(parent.getChildren(), ChildDto.class);
+    }
+
+    public void addChildren(User parent, List<ChildDto> childrenDtoToAdd) {
+        final List<Child> childrenToSave = new ArrayList<>(convertToModel(
+                childrenDtoToAdd, Child.class, ArrayList::new));
+        childrenToSave.forEach(child -> {
+            child.setUser(parent);
+            parent.getChildren().add(child);
+            userRepository.save(parent);
```

Comment:
I did it this way because of children are in set now, and when I did it not in the loop they saving disordered (expected dataset fails). Can you suggest me the proper way to save them in order that I want?